TreasureHunt/Assets/2.Scripts/SimpleCloudHandler.cs

```diff
@@ -93,21 +93,5 @@ public void OnNewSearchResult(TargetFinder.TargetSearchResult targetSearchResult
 				mCloudRecoBehaviour.CloudRecoEnabled = true;
 			}
 		}
-		/*
-		public void OnGUI() {
```

Comment:
One thing you could consider for changes you have in mind but aren't going to do in the current change/pull-request is to leave TODOs for yourself or others.

For example you could write:
// TODO(hyunso): Add code for handling the OnGui state change.